### PR TITLE
Feature/feedback mechanism

### PR DESCRIPTION
**Pull request by iimpulse**
> Adding New Pages
- Link To Us
- Citing Us
- Contact Us
I think these files need to either be vue components that have empty javascript associations or introduce a html loader. But in the essence of time, this will do.

- Shifting items around the navigation
- Simplifying the bottom navigation, adding feedback button

#109 

@monicacecilia has seen and approved these changes.

### CODE REVIEW

File: src/views/Contact.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<div class="container-fluid monarch-view contact">
+    <h2 class="page-title"> Contacting The Monarch Initiative </h2>
+    <div class="contact">
+        <p>
+            If you have a specific question about Monarch, its data, usage, software, or services, please create a 
+            ticket on our <a href="https://github.com/monarch-initiative/helpdesk/issues" target="__blank">issue tracker</a>
```
[the remark this answers is not in the document]
Comment (monicacecilia) [reply src/views/Contact.md]: Alright. just did homework and this was silly.

File: src/components/Navbar.vue [hunk 2]
```diff
@@ -38,33 +37,41 @@
...
+
+          <b-dropdown-item to="/help/links">
+            Link to Us
+          </b-dropdown-item>
+          <b-dropdown-divider/>
 
-        <b-nav-item-dropdown text="Examples">
           <b-dropdown-header class="text-info bg-white">
-            Interesting nodes for Demo or Debugging
+            Examples
```
Comment (monicacecilia) [inline src/components/Navbar.vue]: I would like to remove the the lengthy list of examples from the 'Help' menu. 

@iimpulse you were sharing that vue does not have a way to implement sub-menus under the items listed in the drop-down menus. If we can find a way to implement a submenu under the item 'Examples,' then let's keep it under 'Help.' Else, I suggest that we make a new top-level menu with the word 'Example searches' and take that lengthy list off the 'Help' menu, please.
@pnrobinson would that be okay with you too?

Comment (iimpulse) [reply src/components/Navbar.vue]: These changes have been reverted.

Comment (kshefchek) [reply src/components/Navbar.vue]: it's more that bootstrap doesn't support them, https://github.com/twbs/bootstrap/pull/6342#issuecomment-11594010 but I'm guessing there are hacky ways around it.  Also see https://github.com/bootstrap-vue/bootstrap-vue/issues/3435

Comment (iimpulse) [reply src/components/Navbar.vue]: Yeah.. lets just have it how it is, and if bootstrap vue goes to implement, I'll follow suite. Although when we go to prod I think we should stick to the tags under the search bar

Comment (monicacecilia) [reply src/components/Navbar.vue]: Thank you!

**Pull request merged**